processor/README.md

```diff
@@ -36,14 +36,16 @@ processor documentation for more information.
 
 1. [memory_limiter](memorylimiter/README.md)
 2. *any sampling processors*
+3. Any processor relying on sending source from `Context` (e.g. `k8s_tagger`)
 3. [batch](batchprocessor/README.md)
 4. *any other processors*
 
 ### Metrics
 
 1. [memory_limiter](memorylimiter/README.md)
-2. [batch](batchprocessor/README.md)
-3. *any other processors*
+2. Any processor relying on sending source from `Context` (e.g. `k8s_tagger`)
```

Comment:
Should it be italic as well for consistency?

Reply:
Actually I would even say *Any processor relying on `Context`* because we lose it completely with batching